composer.json

```
@@ -21,7 +21,7 @@
         "illuminate/queue": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
         "illuminate/routing": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
         "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
-        "openzipkin/zipkin": "~1.0",
+        "openzipkin/zipkin": "~2.0",
```

```suggestion
        "openzipkin/zipkin": "~2.0.2",
```

This fixes @KinaneD's issue.